networking/src/p2p/peer.rs

```diff
@@ -370,7 +382,7 @@ async fn bootstrap(msg: Bootstrap, info: Arc<Local>, log: Logger, mut storage: P
         &info.proof_of_work_stamp,
         &Nonce::random().get_bytes(),
         vec![supported_protocol_version.clone()]);
-    storage.store_connection_message(&connection_message, false, addr)?;
+    if store_messages { storage.store_connection_message(&connection_message, false, addr); }
```

Comment:
@fafk @mrjoe7 
I am not a very big fan of this one line IF's :)
what is the rust recommended convention for this?

Reply:
I don't think there is one. I changed it to a multi-line if. :)